### ISSUE TITLE
Unable to set Xbox controller buttons as hotkeys

### ISSUE THREAD

**Issue by vegbrasil** [crmarsh/discord-linux-bugs#104]
Using Discord 0.0.9, Ubuntu 16.04 and no additional drivers for a Xbox 360 USB controller. 

The controller works in games but Discord is unable to use its buttons for hotkey mappings. It works in the Windows version.

**Comment by crmarsh** [comment]
Yeah, I'll add this in at some point.

**Comment by arcooke** [comment]
Is this still on the radar?  

My Logitech F710 doesn't work either, though I suspect there is simply zero gamepad support at this point

Thanks!

**Comment by crmarsh** [comment]
Yeah, you are correct; right now I'm just getting Xinput2 events which only gets keyboard and mouse.
